src/handlers/comment/handlers/query.ts

```diff
@@ -0,0 +1,31 @@
+import { getBotContext, getLogger } from "../../../bindings";
+import { Payload } from "../../../types";
+import { getWalletAddress, getWalletMultiplier } from "../../../adapters/supabase";
+
+export const multiplier = async (body: string) => {
```

Comment:
`multiplier` doesn't make sense for query functionality.

Reply:
i forgot to change that 💀